package.json

```diff
@@ -73,9 +75,9 @@
 		"makepot:android": "npm run wp -- i18n make-pot ./gutenberg/packages --include=*.native.js,*.android.js --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/*,*.js,*.php --subtract=./gutenberg.pot --ignore-domain gutenberg-android.pot",
 		"makepot:ios": "npm run wp -- i18n make-pot ./gutenberg/packages --include=*.native.js,*.ios.js --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/*,*.js,*.php --subtract=./gutenberg.pot --ignore-domain gutenberg-ios.pot",
 		"premakepot:gutenberg": "npm run clean:gutenberg && npm run build:gutenberg",
-		"makepot:gutenberg": "npm run wp -- i18n make-pot ./gutenberg/build --ignore-domain gutenberg.pot",
+		"makepot:gutenberg": "npm run wp -- i18n make-pot ./gutenberg/packages --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/*,*.native.js,*.ios.js,*.android.js,bundle/* --ignore-domain gutenberg.pot",
 		"postmakepot:gutenberg": "npm run clean:gutenberg",
-		"pregenstrings": "test -f gutenberg.pot || npm run makepot:gutenberg",
```

Comment:
The `gutenberg.pot` file can be different if new strings are introduced so I think it's not reliable to prevent generating it if it already exists.